src/WebForms/Configuration/ProvidersHelper.cs

```diff
@@ -0,0 +1,69 @@
+//MIT License
+
+using System.Collections.Specialized;
+using System.Configuration;
+using System.Configuration.Provider;
+using System.Reflection;
+
+namespace System.Web.Configuration
+{
+
+    public static class ProvidersHelper {
```

Comment:
I don't want any of the config stuff brought in. As an initial port, bringing it in as internal is fine but make sure to follow up with removing it